diff --git a/node_modules/@types/buble/index.d.ts b/node_modules/@types/buble/index.d.ts
index 5ac6a96..fa6eea8 100644
--- a/node_modules/@types/buble/index.d.ts
+++ b/node_modules/@types/buble/index.d.ts
@@ -31,6 +31,7 @@ export interface TransformOptions {
     // transforms
     transforms?: {
         arrow?: boolean;
+        asyncAwait?: boolean;
         classes?: boolean;
         collections?: boolean;
         computedProperty?: boolean;
